ntoskrnl/cc/cacheman.c

```diff
@@ -117,22 +117,73 @@ CcShutdownSystem(VOID)
     /* NOTHING TO DO */
 }
 
-/*
- * @unimplemented
- */
+NTKERNELAPI
 LARGE_INTEGER
 NTAPI
-CcGetFlushedValidData (
-    IN PSECTION_OBJECT_POINTERS SectionObjectPointer,
-    IN BOOLEAN BcbListHeld
-    )
+CcGetFlushedValidData(
+    _In_ PSECTION_OBJECT_POINTERS SectionObjectPointer,
+    _In_ BOOLEAN BcbListHeld)
 {
-	LARGE_INTEGER i;
+    PROS_SHARED_CACHE_MAP SharedCacheMap;
+    PROS_VACB Vacb;
+    LARGE_INTEGER Result;
+    LONGLONG LowestDirty = MAXLONGLONG;
+    KIRQL OldIrql;
 
-	UNIMPLEMENTED;
+    //
+    // SectionObjectPointer is NOT optional by annotation.
+    // Only check SharedCacheMap.
+    //
```

Comment:
We don't write multiline comments like that. Check https://reactos.org/wiki/Coding_Style#Commenting.